src/lit-html.ts

```diff
@@ -111,20 +111,24 @@ export function render(
  */
 const attributeMarker = `{{lit-${Math.random()}}}`;
 
-/**
- * Regex to scan the string preceding an expression to see if we're in a text
- * context, and not an attribute context.
- *
- * This works by seeing if we have a `>` not followed by a `<`. If there is a
- * `<` closer to the end of the strings, then we're inside a tag.
- */
-const textRegex = />[^<]*$/;
-const hasTagsRegex = /[^<]*/;
```

Comment:
BTW, this was broken beforehand. Any string passes, including `"<test"`, `"test<"`, and `""`. I'll add a test later tonight.